webui/react/src/routes/utils.ts

```diff
@@ -68,14 +68,18 @@ export const handlePath = (
   if (options.onClick) {
     options.onClick(event);
   } else if (href) {
-    if (event.button === 1 || event.metaKey || event.ctrlKey || options.popout) {
+    if (isShiftClick(event) || options.popout) {
       openBlank(href);
     } else {
       routeAll(href);
     }
   }
 };
 
+export const isShiftClick = (event: MouseEvent|React.MouseEvent): boolean => {
```

Comment:
**suggestion:** hmmm, I wonder if there is a better place for this utility. `utils/browser.ts` or a new one for `utils/event.ts`?